package.json

```diff
@@ -59,6 +59,7 @@
     "@fortawesome/free-solid-svg-icons": "5.14.0",
     "@fortawesome/react-fontawesome": "0.1.8",
     "@hookform/resolvers": "3.3.4",
+    "@tanstack/react-table": "8.20.5",
```

Comment:
is the plan to replace line 104 `"react-table": "6.8.6",` with this? Or do they work independently?

Reply:
Yeah, we want to replace it in the future, but it will probably be in the far-distant future. For now, we would like to have those libraries independently, the new library includes a new "hook" approach which will be needed in the cycle count implementation.

Reply:
sounds good. It's convenient that we can bring in both versions at the same time and then gradually switch over to using this new dependency.

My main concern is around having to support different behaviours across different tables, which may lead to inconsistencies in our UI, but I don't know the react table library well enough to know if that's a real concern or not.

Either way we'd probably need this upgrade eventually, so good that we're starting now